src/data/GraphOnChip.ts

```diff
@@ -659,7 +659,7 @@ export default class GraphOnChip {
                     queue.assignInputs([newChip.createOperand(opName, GraphVertexType.OPERATION)]);
                 } else if (newChip.operationsByName.has(operand.name)) {
                     const op = newChip.operationsByName.get(operand.name);
-                    if (op?.isOffchip) {
+                    if (op?.cores.length === 0) {
```

Comment:
you cant call a property thats actively set and used "uneeded and unimplemented"

you shoudl not transfer operation logic to graph on chip

Reply:
The abstract class defined it and there was one implementation, on other places it was either not implemented or returned `false`, totally defeating the point of having this property in the first place.

And this property was only used in one place, that is why I moved the logic there.